blueprints/ember-frost-icons/index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.0.2'},
+        {name: 'ember-frost-css-core', target: '^1.1.1'}
```

Comment:
Theme is a dependency, but I believe core is only a dev dependency (for the dummy app)